src/test/java/org/folio/permstest/RestVerticleTest.java

```diff
@@ -80,7 +81,7 @@ public static void setup(TestContext context) {
     port = NetworkUtils.nextFreePort();
     TenantClient tenantClient = new TenantClient("http://localhost:" + port, "diku",  null);
     vertx = Vertx.vertx();
-    client = vertx.createHttpClient();
+    client = WebClient.create(vertx);
     DeploymentOptions options = new DeploymentOptions().setConfig(new JsonObject()
         .put("http.port", port).put(PermsCache.CACHE_HEADER, false)).setWorker(true);
     try {
```

Comment:
Since RMB 26 RMB automatically starts embedded postgres. Please remove this try-catch block.
This will also allow developers to set some DB_ environment variables to be used for tests.
The same for RestVerticleWithCacheTest.

Reply:
We cannot omit try here, as public void startEmbeddedPostgres() throws IOException. Did I misunderstand your suggestion?

> This will also allow developers to set some DB_ environment variables to be used for tests.
What should be done for it in the code?

Reply:
Fixed.